lib/std/build.zig

```diff
@@ -2888,6 +2889,48 @@ pub const LibExeObjStep = struct {
 
         try zig_args.append("--enable-cache");
 
+        const is_build = switch (self.kind) {
+            .lib => true,
+            .exe => true,
+            .obj => true,
+            else => false,
+        };
+        if (is_build) {
```

Comment:
Why would this not apply to `test`?

Reply:
Ah, good point, it should. Not sure why I did that.